tests/core/test_hydra_cli.py

```diff
@@ -0,0 +1,34 @@
+from __future__ import annotations
+
+import sys
+
+import hydra
+import pytest
+
+from fairchem.core._cli import main
+from fairchem.core.common import distutils
+
+
+def test_hydra_cli():
+    distutils.cleanup()
```

Comment:
This is a nit, i dont know how to best do this. Would a method scoped fixture make sense for this?

Reply:
im just trying to avoid fancy pytest syntax :), i dont know if fixture is the right thing i think if theres alot of these we can just move it to a setup() call?